sub-packages/bionemo-geometric/pyproject.toml

```diff
@@ -19,6 +19,7 @@ dependencies = [
     'torch-geometric==2.5.0',
     'torch-scatter==2.1.2',
     'torch_sparse==0.6.18',
+    'rdkit',
```

Comment:
Please provide a version for this dependency. Please either pin it (`==`) or provide a range of compatible values.